lib/nextstrain/sphinx/theme/static/css/nextstrain.css

```diff
@@ -3,27 +3,41 @@
 @import "theme.css"; /* From sphinx_rtd_theme */
 
 html {
-  --text-color: black;
+  --text-color: #24292e;
   --heading-color: #404040;
   --link-color: #5097ba;
   --sidebar-background-color: #f2f2f2;
+  --content-background-color: #ffffff;
 }
 
 body {
   font-family: Lato, 'Helvetica Neue', sans-serif;
-  font-weight: 300;
+  font-weight: 400;
   color: var(--text-color);
+  line-height: 1.5;
 }
 
 h1, h2, h3, h4, h5, h6, legend, .rst-content .toctree-wrapper p.caption {
   font-family: Lato, 'Helvetica Neue', sans-serif;
-  font-weight: 400;
+  font-weight: 600;
   color: var(--heading-color);
 }
 
+p {
+  line-height: inherit;
+}
+
 a {
   color: var(--link-color);
 }
+/* don't use a new colour (default is purple) for visited links */
+a:visited {
+  color: var(--link-color);
+}
```

Comment:
I would much prefer to keep visited links colored differently than unvisited ones. It's an accessibility feature and navigational marker that's well studied: https://www.nngroup.com/articles/change-the-color-of-visited-links/ (just one example). I also find myself using it, particularly in documentation hierarchies, to find my way back to a page thru a series of clicks.

Reply:
No worries -- removed these 4 lines in 3b2302f